### PR TITLE
199

### PR DESCRIPTION
**Pull request by smitaraut**
## Summary of Changes
Changed plugin folder name to match the driver name spectrumscale.csi.ibm.com
Also deleting any stale plugin folder with old name ibm-spectrum-scale-csi

## Unit Tests
- Plugin folder is created with name spectrumscale.csi.ibm.com and ibm-spectrum-scale-csi folder is deleted if it exists
- PVCs and PODs created before continue to be accessible
- New PVCs and PODs can be created

## Documentation Updates
None

## Upgrade Considerations
Old plugin folder should get deleted on upgrade

## Linked Issues
#199

### CODE REVIEW

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/files/deploy/crds/csiscaleoperators.csi.ibm.com.cr.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../../../../operator/ibm-spectrum-scale-csi-operator/deploy/crds/csiscaleoperators.csi.ibm.com.cr.yaml
```
Comment (mew2057) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/files/deploy/crds/csiscaleoperators.csi.ibm.com.cr.yaml:1]: Please revert this  change, it's going to cause issues with CASE cert.

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperatorSetup/files/deploy/crds/csiscaleoperators.csi.ibm.com.crd.yaml [hunk 2]
```diff
@@ -1 +1 @@
-../../../../../../../operators/ibm-spectrum-scale-csi-operator/deploy/crds/csiscaleoperators.csi.ibm.com.crd.yaml
\ No newline at end of file
+../../../../../../../operator/ibm-spectrum-scale-csi-operator/deploy/crds/csiscaleoperators.csi.ibm.com.crd.yaml
```
Comment (mew2057) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperatorSetup/files/deploy/crds/csiscaleoperators.csi.ibm.com.crd.yaml:1]: Please revert this  change, it's going to cause issues with CASE cert.

File: driver/cmd/ibm-spectrum-scale-csi/main.go [hunk 3]
```diff
@@ -71,3 +75,11 @@ func createPersistentStorage(persistentStoragePath string) error {
 	}
 	return nil
 }
+
+func deleteStalePluginDir(stalePluginPath string) error {
+        if err := os.RemoveAll(stalePluginPath); err != nil {
```
Comment (mew2057) [inline driver/cmd/ibm-spectrum-scale-csi/main.go:80]: As a style point, can't we just return err and not have a branching conditional for the return?

File: driver/csiplugin/gpfs.go [hunk 4]
```diff
@@ -32,7 +32,8 @@ import (
 
 // PluginFolder defines the location of scaleplugin
 const (
-	PluginFolder          = "/var/lib/kubelet/plugins/ibm-spectrum-scale-csi"
+	PluginFolder          = "/var/lib/kubelet/plugins/spectrumscale.csi.ibm.com"
+        OldPluginFolder       = "/var/lib/kubelet/plugins/ibm-spectrum-scale-csi"
```
Comment (mew2057) [inline driver/csiplugin/gpfs.go:36]: Not a major  issue, but the spacing here is irregular.

**Comment by smitaraut** [comment]
Closing this PR as it was created from a branch with merge issues. Review comments have been incorporated in new PR #208

**Pull request closed without merging**